config/locales/client.en.yml

```diff
@@ -380,7 +380,11 @@ en:
     send_invite: 'Send {{count}} invites'
     to: 'to: '
     default_message: "Hey there, come make decisions with us! We're using a collaborative decision-making tool called Loomio"
-
+    messages:
+      members_added_and_emails_sent: '{{members}} member(s) added and {{emails}} invitation(s) sent'
+      members_added: '{{members}} member(s) added'
+      emails_sent: '{{emails}} invitation(s) sent'
+      no_invitations: 'No invitation(s) sent!'
```

Comment:
Will we need the "no invitations sent" case? I think our client will be smart enough to prevent this happening. IE: I imagine we'll have client side email validation and disable send button until the form is valid.